.travis.yml

```diff
@@ -5,7 +5,7 @@ dist: focal
 language: node_js
 
 node_js:
-  - 14.21.2
+  - 16.20.1
```

Comment:
So, `16.20.1` is the supported version of choice for the other Node.js repos?

Reply:
It was for the export service that was recently upgraded https://github.com/tidepool-org/export/blob/94988ea6eda903853967cba86942d6ac6b839d98/.travis.yml#L6